### PR TITLE
Amazon SQS Connector Reference Guide - 5-5-0

### PR DESCRIPTION
**Pull request by dhirsch1**

### CODE REVIEW

File: modules/ROOT/pages/amazon/amazon-sqs-connector-reference.adoc [hunk 1]
```diff
@@ -772,21 +776,92 @@ Sets the value of one or more queue attributes. When you change a queue's attrib
...
+
+[[crl-file]]
+=== CRL File
+
+[cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
+|======================
+| Field | Type | Description | Default Value | Required
+| Path a| String | The path to the CRL file. |  |
+|======================
 [[Reconnection]]
```
Comment (valkyrie69) [inline modules/ROOT/pages/amazon/amazon-sqs-connector-reference.adoc:856]: Replace the reconnect block per our doc

**Review by valkyrie69 — APPROVED** [review]
Just replace the reconnect block per our process doc. Otherwise, looks good.

File: modules/ROOT/pages/amazon/amazon-sqs-connector-reference.adoc (hunk 1 above)
Comment (dhirsch1) [reply modules/ROOT/pages/amazon/amazon-sqs-connector-reference.adoc:856]: Hmm - I did that for Reconnect. You can see that below if you expand the text.

**Review by valkyrie69 — APPROVED** [review]
LGTM

**Pull request merged**